### DISCUSSION TITLE
Security onion Manager 2.4 stuck on starting soc

### DISCUSSION THREAD

**Discussion by mohasmr** [Security-Onion-Solutions/securityonion#13158] [2.4]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

16

### Storage for /

71

### Storage for /nsm

137

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

after $so-soc-start --force the security onion stuck on 
""System appears to be starting. No highstate has completed since the system was restarted""

i was checking the salt and everything is ok 
**$sudo systemctl status salt-master**
● salt-master.service - The Salt Master Server
     Loaded: loaded (/usr/lib/systemd/system/salt-master.service; enabled; preset: disabled)
     Active: active (running) since Fri 2024-06-07 07:10:08 UTC; 55min ago
       Docs: man:salt-master(1)
             file:///usr/share/doc/salt/html/contents.html
             https://docs.saltproject.io/en/latest/contents.html
   Main PID: 901 (/opt/saltstack/)
      Tasks: 39 (limit: 99156)
     Memory: 659.9M
        CPU: 4min 4.998s
     CGroup: /system.slice/salt-master.service
             ├─ 901 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master MainProcess"
             ├─2132 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master PubServerChannel._publish_daemon"
             ├─2133 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master EventPublisher"
             ├─2138 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master Engine(salt.loaded.ext.engines.checkmine)"
             ├─2140 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master Maintenance"
             ├─2141 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer ReqServer_ProcessManager"
             ├─2142 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorkerQueue"
             ├─2143 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorker-0"
             ├─2144 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorker-1"
             ├─2146 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorker-2"
             ├─2148 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master FileServerUpdate"
             ├─2155 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorker-3"
             └─2157 "/opt/saltstack/salt/bin/python3.10 /usr/bin/salt-master ReqServer MWorker-4"

Jun 07 08:04:12 manager salt-master[2138]: [INFO    ] Runner completed: 20240607080411838975
Jun 07 08:05:12 manager salt-master[2138]: [INFO    ] Runner completed: 20240607080512791066
Jun 07 08:05:13 manager salt-master[2138]: [INFO    ] Runner completed: 20240607080513176591
Jun 07 08:05:14 manager salt-master[2138]: [INFO    ] Runner completed: 20240607080514369823
Jun 07 08:05:15 manager salt-master[2143]: [INFO    ] Authentication request from manager_manager
Jun 07 08:05:15 manager salt-master[2143]: [INFO    ] Authentication accepted from manager_manager
Jun 07 08:05:16 manager salt-master[2155]: [INFO    ] Runner completed: 20240607080516887803
Jun 07 08:05:17 manager salt-master[2155]: [INFO    ] Runner completed: 20240607080517325794
Jun 07 08:05:18 manager salt-master[2155]: [INFO    ] Runner completed: 20240607080518093632
Jun 07 08:05:18 manager salt-master[2155]: [INFO    ] Runner completed: 20240607080518509504

next i try **$sudo so-checkin** and there is no error

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
> ""System appears to be starting. No highstate has completed since the system was restarted""

How long did you wait for this message to go away?

Have you tried rebooting?

**Comment by mohasmr** [comment]
it was an error from the less of physical ressources
